Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/kubernetes-nmstate
```

Comment:
Despite the operator will be forked under openshift, in this upstream repo it should remain agnostic. Please s/openshift/nmstate.